src/main/groovy/nebula/plugin/dependencylock/tasks/GenerateLockTask.groovy

```diff
@@ -85,27 +89,35 @@ class GenerateLockTask extends AbstractLockTask {
         if (DependencyLockTaskConfigurer.shouldIgnoreDependencyLock(project)) {
             throw new DependencyLockException("Dependency locks cannot be generated. The plugin is disabled for this project (dependencyLock.ignore is set to true)")
         }
-        Collection<Configuration> confs = getConfigurations() ?: lockableConfigurations(project, project, getConfigurationNames())
+        Collection<Configuration> confs = getConfigurations() ?: lockableConfigurations(project, project, getConfigurationNames(), getSkippedConfigurationNames())
         Map dependencyMap = new GenerateLockFromConfigurations().lock(confs)
         new DependencyLockWriter(getDependenciesLock(), getSkippedDependencies()).writeLock(dependencyMap)
     }
 
-    static Collection<Configuration> lockableConfigurations(Project taskProject, Project project, Set<String> configurationNames) {
+    static Collection<Configuration> lockableConfigurations(Project taskProject, Project project, Set<String> configurationNames, Set<String> skippedConfigurationNamesPrefixes = []) {
```

Comment:
Should this be using `DEFAULT_NON_LOCKABLE_CONFIGURATIONS` as the default `skippedConfigurationNamesPrefixes`, @rpalcolea ?

Reply:
I thought about that, but then changed my mind as other people might not want default for skipping

Reply:
👍